src/raft_server.c

```diff
@@ -1124,54 +1124,53 @@ void raft_offer_log(raft_server_t* me_, raft_entry_t* entries,
     }
 }
 
-void raft_pop_log(raft_server_t* me_, raft_entry_t* ety, const int idx)
+void raft_pop_log(raft_server_t* me_, raft_entry_t* entries,
+                    const int n_entries, const int idx)
 {
     raft_server_private_t* me = (raft_server_private_t*)me_;
+    int i;
 
-    if (me == NULL || ety == NULL)
+    if (me == NULL || entries == NULL)
         return;
 
-    if (!raft_entry_is_cfg_change(ety))
-        return;
+    for (i = 0; i < n_entries; i++) {
+        raft_entry_t *ety = &entries[-i];
```

Comment:
Took me a while to understand this... Would it be better to follow the normal convention for entries[n_entries]?